vrf/src/alias.rs

```diff
@@ -1,31 +1,13 @@
 #![allow(non_snake_case)]
 
-use std::{
-    cmp::{Ord, Ordering, PartialOrd},
-    fmt::Debug,
-    ops::{Add, Mul, Sub},
-};
-
-use num_traits::{sign::Unsigned, FromPrimitive, ToPrimitive};
+use std::cmp::{Ord, Ordering};
 
 use crate::rng::Rng;
 
-pub struct AliasMethod<P>
-where
-    P: Copy
-        + Debug
-        + Unsigned
-        + Add<P>
-        + Sub<P>
-        + Mul<P>
-        + FromPrimitive
-        + ToPrimitive
-        + PartialOrd<P>
-        + Ord,
-{
+pub struct AliasMethod {
```

Comment:
Could you add some `rustdoc`?

Reply:
Done, and renamed it to something clearer to me.